package.json

```diff
@@ -29,7 +30,8 @@
     "version": "bash scripts/version.sh && git add .",
     "version-nightly": "bash scripts/should-nightly-run.sh && npm version prerelease --no-git-tag-version --preid=$(git rev-parse HEAD)",
     "postversion": "bash scripts/push-if-git.sh",
-    "bundlesize": "yarn build && yarn build:dead-code-elimination && size-limit"
+    "bundlesize": "yarn build && yarn build:dead-code-elimination && size-limit",
+    "postinstall": "npx yarn-deduplicate"
```

Comment:
I was getting type errors when I ran this in the repo earlier. I thought it broke some of the packages.

Reply:
yeah i fixed em